<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>431</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>437</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>-8</y>
     <width>711</width>
     <height>441</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="img.qrc">:/icons/img/Siemens_AISIN_Capital_.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>441</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/icons/img/service.png</normaloff>:/icons/img/service.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Update</string>
    </attribute>
    <widget class="QLabel" name="imglabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-2</y>
       <width>701</width>
       <height>431</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/icons/img/Siemens_AISIN_Capital_.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>320</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar{
border-radius:10px;
background-color: rgb(0, 10, 25);
border:2px solid  #dedede;
}
QProgressBar::chunk{
border-radius:10px;
background-color: rgb(120, 120, 120);
}</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="discription_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>681</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 24pt &quot;Arial Narrow&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Please, wait a minute the system is being updated now.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="FOTA1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>661</width>
       <height>86</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>48</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 48pt &quot;Palatino Linotype&quot;;
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>FOTA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/icons/img/idea.png</normaloff>:/icons/img/idea.png</iconset>
    </attribute>
    <attribute name="title">
     <string>History</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>-4</x>
       <y>2</y>
       <width>701</width>
       <height>401</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="imglabel_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>431</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/icons/img/Siemens_AISIN_Capital_.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>691</width>
       <height>271</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>Update1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Update2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Update3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Update4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Update5</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Update details</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
